plugins/woocommerce-blocks/assets/js/blocks/cart/block.js

```diff
@@ -26,11 +26,19 @@ import './style.scss';
 const reloadPage = () => void window.location.reload( true );
 
 const Cart = ( { children, attributes = {} } ) => {
-	const { cartIsLoading } = useStoreCart();
+	const cart = useStoreCart();
 	const { hasDarkControls } = attributes;
 
+	const { dispatchStoreEvent } = useStoreEvents();
```

Comment:
despite the misleading name, for Mini Cart/Cart/Checkout, you should always use `dispatchCheckoutEvent` in those contexts, it already include the Cart in its response.

Reply:
I switched to `dispatchCheckoutEvent` in all cases besides `calculate-shipping`, but when I started adding docs, it made me think that now there are some cart events without the checkout prefix, and other events taht do have the checkout prefix. For example:

`experimental__woocommerce_blocks-cart-remove-item` 
vs
`experimental__woocommerce_blocks-checkout-mini-cart-open`

Maybe this is fine, but I just wanted to point it out.

Reply:
Yeah the Checkout prefix might have been a mistake, we will revisit the names of all of those events once we graduate them.